components/automate-ui/src/app/entities/projects/project.effects.ts

```diff
@@ -203,8 +205,11 @@ export class ProjectEffects {
   getLatestApplyRulesStatus$ = observableInterval(1000 * POLLING_INTERVAL_IN_SECONDS).pipe(
     withLatestFrom(this.store.select(iamMajorVersion).pipe(filter(identity))),
     withLatestFrom(this.store.select(iamMinorVersion).pipe(filter(identity))),
-    switchMap(([[_, major], minor]) => {
-      if (major === 'v2' && minor === 'v1') {
+    withLatestFrom(this.store.select(allPerms)),
+    switchMap(([[[_, major], minor], perms]) => {
```

Comment:
what's the `_` stand for here?

Reply:
It just means we are ignoring that value I think.

Reply:
I meant what would that value be if it weren't ignored?

Reply:
The four params are timer_tick, major, minor, and perms; it is not two values for major.